source/auth/auth.rst

```diff
@@ -1270,9 +1270,9 @@ mechanism
     MUST be "MONGODB-OIDC"
 
 mechanism_properties
-    DEVICE_NAME
-        Drivers MUST allow the user to specify a name for the device
-        workflow that is one of "aws", "azure", or "gcp".
+    SERVICE_NAME
+        Drivers MUST allow the user to specify a name for using a service
+        to obtain credentials that is one of "aws", "azure", or "gcp".
```

Comment:
This has very different semantics than it does in the GSSAPI case – not to make your life harder, but maybe it’s a good idea to keep using a different name (suggestion off the top of my head: `PROVIDER_NAME`)?

Reply:
That was @blink1073 's original suggestion - I thought it was a bit too generic but since you do point out the Kerberos meaning I'm fine with changing it to PROVIDER_NAME. Thoughts @blink1073 ?

Reply:
`PROVIDER_NAME` works for me, it is generic enough for us to use a non-cloud provider in the future.

Reply:
I've updated to `PROVIDER_NAME`